lib/install/management/delete_test.go

```diff
@@ -90,8 +90,8 @@ func TestDelete(t *testing.T) {
 		// FIXME: cannot check if it's VCH or not
 		// testNewVCHFromCompute(input.ComputeResourcePath, input.DisplayName, validator, t)
 		//	testDeleteVCH(validator, conf, t)
```

Comment:
Looks like we need to implement SearchIndex in vcsim for that to work, so I'll leave it for now.